website/docs/language/resources/ephemeral/write-only.mdx

```diff
@@ -167,4 +171,30 @@ In the above example, the ephemeral resource `aws_secretsmanager_secret_version`
 
 Terraform first creates the secret in AWS Secrets Manager using the ephemeral `random_password`, then retrieve it using the ephemeral `aws_secretsmanager_secret_version` resource, and finally write the password to the write-only `password_wo` argument of the `aws_db_instance` resource.
 
+### Create a MySQL database in Azure
+
+Another example of a using a write-only argument is to create a MySQL database in Azure. In this example, Terraform generates an password using an `ephemeral` resource and passes it to the `administrator_password_wo` write-only argument of the `azurerm_mysql_flexible_server` resource:
```

Comment:
```suggestion
You can use a write-only argument to create a MySQL database in Azure. In the following example, Terraform generates an password using an `ephemeral` resource and passes it to the `administrator_password_wo` write-only argument of the `azurerm_mysql_flexible_server` resource:
```
If someone just scanned and started reading this example, "Another example . . . " would be jarring. We should avoid assuming that someone is reading the page linearly.